scripts/scheduler.py

```diff
@@ -375,9 +376,12 @@ def queue_new_jobs(available_slots, env, config, studies, wes_token, exclude_stu
           graphql_url = config['compute_environments'][env]['graphql_url']
           try:
               run_info = get_run_state(graphql_url, latest_run_id, wes_token)
-              resume = run_info['sessionId'] if run_info.get('sessionId') else False
-              work_dir = run_info['engineParameters']['workDir']
-              launch_dir = run_info['engineParameters']['launchDir']
+              if run_info.get('sessionId') and re.match("^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$", run_info.get('sessionId')):
```

Comment:
the second `run_info.get('sessionId')` can be changed to `run_info.get('sessionId', '')`, this will prevent runtime exception when `run_info.get('sessionId')` returns `None`.